files/init-container.sh

```diff
@@ -9,22 +9,22 @@ LOCAL_SRPM_DIR=$HOME/local-SRPMs
 
 case $GIT_BRANCH in
 	trunk-pvs-direct)
-		XS_REPO_KEY=aHR0cDovL3hzLXl1bS1yZXBvcy5zMy13ZWJzaXRlLXVzLWVhc3QtMS5hbWF6b25hd3MuY29tLzhlZmIzZWUtM2M3ZC0xMWU2LWI5NWUtMzNkZTkzOTJkZGYyL2RvbWFpbjA=
+		XS_REPO_KEY=aHR0cDovL3hhcGktcmVwb3MuczMtd2Vic2l0ZS11cy1lYXN0LTEuYW1hem9uYXdzLmNvbS84ZWZiM2VlLTNjN2QtMTFlNi1iOTVlLTMzZGU5MzkyZGRmMi9kb21haW4w
```

Comment:
As a side-note, RPMs for this branch are no longer uploaded, so could be removed?